clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -720,6 +733,22 @@ private void maybeOverrideEnableAutoCommit(Map<String, Object> configs) {
         }
     }
 
+    private void maybeOverrideGroupRemoteAssignorAndLocalAssignors(Map<String, Object> configs) {
+        if (getString(GROUP_PROTOCOL_CONFIG).equals(DEFAULT_GROUP_PROTOCOL)) {
+            // overwrite in case of group protocol is classic
+            if (getString(GROUP_REMOTE_ASSIGNOR_CONFIG) != null && !getString(GROUP_REMOTE_ASSIGNOR_CONFIG).isEmpty()) {
+                configs.put(GROUP_REMOTE_ASSIGNOR_CONFIG, DEFAULT_GROUP_REMOTE_ASSIGNOR);
+            }
+            if (!getList(GROUP_LOCAL_ASSIGNORS_CONFIG).isEmpty()) {
+                configs.put(GROUP_LOCAL_ASSIGNORS_CONFIG, Collections.emptyList());
+            }
```

Comment:
As mentioned above, I think code related to the local assignors should be omitted for now.

Reply:
Yes, remove `GROUP_LOCAL_ASSIGNORS_CONFIG`. Thank you.